CONTRIBUTING.md

```diff
@@ -247,6 +247,61 @@ The tags used in the commit message shall follow the same scheme as the tags
 present in the `WIP` branch names, described in the
 [Branching Scheme](#branching-guidelines).
 
+### Sign your work
+
+Signing your work involves adding a line at the end of your patch's git commit
+messages. Your sign-off certifies that either you created the patch or have the
+right to submit it under an appropriate open source license, as described in
+the certificate below (from [developercertificate.org](http://developercertificate.org/)):
+
+```
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+1 Letterman Drive
+Suite D4700
+San Francisco, CA, 94129
+
+Everyone is permitted to copy and distribute verbatim copies of this
+license document, but changing it is not allowed.
+
+
+Developer's Certificate of Origin 1.1
+
+By making a contribution to this project, I certify that:
+
+(a) The contribution was created in whole or in part by me and I
+    have the right to submit it under the open source license
+    indicated in the file; or
+
+(b) The contribution is based upon previous work that, to the best
+    of my knowledge, is covered under an appropriate open source
+    license and I have the right under that license to submit that
+    work with modifications, whether created in whole or in part
+    by me, under the same open source license (unless I am
+    permitted to submit under a different license), as indicated
+    in the file; or
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+Then to sign your work, add this line to each of your git commit messages using
+your real name:
+
+```
+Signed-off-by: Jane Doe <jane.doe@email.com>
+```
+Alternatively, if `user.name` and `user.email` is set in git configs, you can
```

Comment:
If your `user.name` and `user.email` are set in our git configs, you can sign each commit automatically by using the `git commit -s` command.